src/environments/environment.ts

```diff
@@ -10,4 +10,5 @@
 
 export const environment = {
   production: false,
+  socketio_url: 'http://pcitdb88:3000', //TO ADAPT
```

Comment:
For the development environment this can be set to **localhost**, where the development backend typically runs.
This is a good default which will work in most of our dev setups

Reply:
This parameter needs to be defined also for the production environment.
If would be nice to have it somehow externally configurable (To avoid hardcoding values)